bin/rucio

```diff
@@ -1489,6 +1490,42 @@ def list_rses(args):
     return SUCCESS
 
 
+@exception_handler
+def list_suspicious_replicas(args):
+    """
+    %(prog)s list-suspicious-replicas [options] <field1=value1 field2=value2 ...>
+
+    List replicas marked as suspicious.
+
+    """
+    client = get_client(args)
+    rse_expression = None
+    younger_than = None
+    nattempts = None
+    if args.rse_expression:
+        rse_expression = args.rse_expression
+    if args.younger_than:
+        younger_than = args.younger_than
+    if args.nattempts:
+        nattempts = args.nattempts
+    # Generator is a list with one entry, which itself is a list of lists.
+    replicas_gen = client.list_suspicious_replicas(rse_expression, younger_than, nattempts)
+    for i in replicas_gen:
+        replicas = i
+
+    data = [['', '', '', '', ''], ['RSE Expression:', 'Scope:', 'Created at:', 'Nattempts:', 'File Name:'], ['', '', '', '', '']]
+    for rep in replicas:
+        data.append([rep['rse'], rep['scope'], rep['created_at'], rep['cnt'], rep['name']])
+    col_width_0 = max(len(str(row[0])) for row in data) + 6
+    col_width_1 = max(len(str(row[1])) for row in data) + 6
+    col_width_2 = max(len(str(row[2])) for row in data) + 6
+    col_width_3 = max(len(str(row[3])) for row in data) + 6
+    for row in data:
```

Comment:
Why not use `tabulate` here?

Reply:
I hadn't heard of tabulate before and this was a simple solution that makes the output easy to read.

Reply:
Please change it to tabulate. It will produce the same output, but in that case all the commands use the same workflow, which makes it easier to read.